project-repo/review-process.md

```diff
@@ -0,0 +1,74 @@
+* Any commits merged in the `main` branch need to pass the tests. This is our mainline. You can split your changes into multiple, more discrete modifications, so as to help give additional context when one needs to dig into the git history later on.
+* Reviewing a change is serious business. A generic "LGTM" should be more the exception than the rule. Comments are encouraged and should be seen as positive feedback to broaden our understanding of the issue at hand. Accordingly, the commenter should strive to make their feedback as constructive as possible.
+* When there are multiple reviewers on a PR, please pay attention to the comments and answers of your peers!  This helps building shared understanding of the code. All of us could always learn something new. :-)
+* Try to not let an open pull request remain without activity for more than 3 days. Especially, try to be responsive within a day for contributions from external contributors. Stale branches should be either worked on or deleted. This helps ensure that the repository has an easy and actionable pull request review list.
+* Automated pull requests for dependency updates should be looked at with scrutiny: tests passing does not necessarily imply that the pull request is good to merge. Examine at the upstream changelog and/or commit logs, and assess the impact to see if we would benefit from the changes, and generally aim to reduce technical debt. If any changes are required to make the tests pass again for such a pull request, those changes should be added to the very same branch, so that merging the pull request would not leave the tests broken.
+
+### Merge workflow
+
+We should use the **merge commit workflow** (not squash and merge). The history of each branch should make sense on its own, with few or no "fixup" commits.
+
+However, for better readability of the mainline, the merge commit in the mainline should follow [conventional commits][convcommits]. This can be enforced [on a per project][gh-docs-merge-commit] basis on GitHub.
+
+> TODO: have a document defining what to configure in every new GitHub project (webhook for Jira, merge commit + default commit template, automated merged branch deletion).
+
+[convcommits]: https://www.conventionalcommits.org/en/v1.0.0/
+[gh-docs-merge-commit]: https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/configuring-pull-request-merges/configuring-commit-merging-for-pull-requests
+
+## Process
+
+Don't hesitate to comment and reiterate! When you engage with a pull request, the expectation is that you remain responsive, and not let the pull request go stale.
+
+1. The pull request should always be opened as a draft, to not notify the reviewers from automation. Ensure the description of changes is readable and understandable by prospective reviewers. The pull request should be linked to a Jira card, using `UDENG-XXX` in its description, so that Jira links against it.
```

Comment:
```suggestion
1. The pull request should always be opened as a draft, to avoid notitying the reviewers before the tests pass and the pull request is fully ready. Ensure the description of changes is readable and understandable by prospective reviewers. The pull request should be linked to a Jira card, using `UDENG-XXX` in its description, so that Jira links against it.
```